src/Microsoft.TemplateEngine.Abstractions/IPostAction.cs

```diff
@@ -8,14 +8,29 @@ namespace Microsoft.TemplateEngine.Abstractions
 {
     public interface IPostAction
     {
+        /// <summary>
+        /// Gets the description of the post action.
+        /// </summary>
         string Description { get; }
 
+        /// <summary>
+        /// Gets the identifier of the action that will be performed.
+        /// Note that this is not an identifier for the post action itself.
+        /// </summary>
         Guid ActionId { get; }
 
+        /// <summary>
+        /// Gets a value indicating wheather the template instantiation should continue
+        /// in case of an error with this post action.
+        /// </summary>
         bool ContinueOnError { get; }
 
         IReadOnlyDictionary<string, string> Args { get; }
 
+        /// <summary>
+        /// Gets the instructions that should be manually performed by the user
+        /// as part of this post action.
```

Comment:
Nit: Add comment, that this is used when host doesn't have post action processor implemented.

Reply:
Even if it is only implemented by the CLI today, we have this _property_ in the interface under Abstractions assembly. As I understand it, that is the purpose of `ManualInstructions`. The comment only explains the situation that is true regardless. That's why I think it makes sense to have it here.

Reply:
Imo, its VS bug to not inform user to do something manually if VS didn't do it for them... VS being Windows app, would show dialog...